src/common/log.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import log, {LevelOption, LogFunctions} from 'electron-log';
+
+import Util from 'common/utils/util';
+
+const logFunctions = {
```

Comment:
0/5 Maybe this could be a class instead of an object? It feels like that might make storing and applying the prefix a bit more straightforward, but that might not work with electron-log's interface

Reply:
Yeah, I'm not sure why it's not a class. I will make it a class.